From 560b0aeba9277c626c7dbc172d379f0e42e7eced Mon Sep 17 00:00:00 2001
From: Tsz-Wo Nicholas Sze <szetszwo@hortonworks.com>
Date: Tue, 19 Dec 2017 10:02:30 +0800
Subject: [PATCH 2732/2848] HDFS-12347.
 TestBalancerRPCDelay#testBalancerRPCDelay fails
 very frequently.  Contributed by Bharat
 Viswanadham

(cherry picked from commit c4e58ea8234ba41f250e2c2663088986c6d48c35)

Change-Id: If0a6d25ecb0dbf61ebeee5a0bc4d0c197707be56
---
 .../hadoop/hdfs/server/balancer/TestBalancer.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/balancer/TestBalancer.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/balancer/TestBalancer.java
index 9eb5f88..260256e 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/balancer/TestBalancer.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/balancer/TestBalancer.java
@@ -1739,7 +1739,7 @@ void testBalancerRPCDelay() throws Exception {
     initConf(conf);
     conf.setInt(DFSConfigKeys.DFS_BALANCER_DISPATCHERTHREADS_KEY, 30);
 
-    int numDNs = 40;
+    int numDNs = 20;
     long[] capacities = new long[numDNs];
     String[] racks = new String[numDNs];
     for(int i = 0; i < numDNs; i++) {
-- 
1.7.9.5

